code/modules/shield_generators/shield_generator.dm

```diff
@@ -14,7 +14,7 @@
 	var/datum/wires/shield_generator/wires
 	var/list/field_segments = list()	// List of all shield segments owned by this generator.
 	var/list/damaged_segments = list()	// List of shield segments that have failed and are currently regenerating.
-	var/list/event_log() = list()				// List of relevant events for this shield
```

Comment:
oh hey this is my code. Those brackets were a typo, not sure why i never noticed it.

Reply:
You didn't notice because DM somehow has no problem parsing this mess.